lwt.opam

```diff
@@ -43,8 +43,6 @@ conflicts: [
 ]
 
 build: [
-  ["dune" "exec" "-p" name "src/unix/config/discover.exe" "--" "--save"
-    "--use-libev" "%{conf-libev:installed}%"]
```

Comment:
This is a problem when cross compiling from Linux desktop to any non libev platform, like Android or iOS.

But it shouldn't be needed as it is enabled by default if it is not Windows or Android

https://github.com/ocsigen/lwt/blob/master/src/unix/config/discover.ml#L417